exchanges/bittrex/bittrex.go

```diff
@@ -408,6 +409,10 @@ func (b *Bittrex) SendAuthHTTPRequest(ep exchange.URL, method, action string, pa
 		HTTPDebugging:  b.HTTPDebugging,
 		HTTPRecording:  b.HTTPRecording,
 		HeaderResponse: resultHeader,
+	}
+
+	return b.SendPayload(context.Background(), request.Unset, func() (*request.Item, error) {
```

Comment:
This function also uses timestamp based signature and headers. I would advise inlining. Bittrex has an unimplemented rate limit (not in scope, but relevant)